goben/client.go

```diff
@@ -310,3 +337,27 @@ func workLoop(conn, label, cpsLabel string, f call, buf []byte, reportInterval t
 
 	acc.average(start, conn, label, cpsLabel)
 }
+
+func validateProberConfig(probeInterval, pktInterval string, pktPerProbe int) (time.Duration, time.Duration, int) {
+	if len(probeInterval) < 1 || len(pktInterval) < 1 || pktPerProbe < 1 {
+		// if the input values are invalid, then return default values
+		log.Println("WARNING: Invalid prober configuration. Reset to default values.")
+		probeInterval = "3s"
+		pktInterval = "500ms"
+		pktPerProbe = 3
```

Comment:
This is not a good way to reset to default values. You will have to manage this twice in the code now. Why not just exit the program with an error?

Reply:
done